contracts/vaults/base/RibbonVault.sol

```diff
@@ -567,13 +567,21 @@ contract RibbonVault is OptionsVaultStorage {
         view
         returns (uint256)
     {
-        uint8 decimals = vaultParams.decimals;
-        uint256 numShares = shares(account);
+        uint256 _decimals = vaultParams.decimals;
         uint256 pps =
-            totalBalance().sub(vaultState.totalPending).mul(10**decimals).div(
-                totalSupply()
+            ShareMath.pricePerShare(
+                totalSupply(),
+                totalBalance(),
+                vaultState.totalPending,
+                _decimals
+            );
+
+        return
+            ShareMath.sharesToUnderlying(
+                shares(account),
+                pps,
+                uint8(_decimals)
```

Comment:
[This PR](https://github.com/ribbon-finance/ribbon-v2/pull/88) needs to be merged before L04. We're casting to uint8 because we're changing the parameter to uint256 in a separate PR

Will remove the cast after that's merged